### DISCUSSION TITLE
Migration failing with 'Ray seems not be running at http://10.0.2.2:23517'

### DISCUSSION THREAD

**Discussion by mattostromhall** [spatie/laravel-ray#88] [Q&A] [answered]
Hello :) 

I am using a database on an external host from my local environment. My local environment is running through Homestead. However, when I try to run a migration it errors with 'Ray seems not be running at http://10.0.2.2:23517'. Before I installed Ray migrations were running without issue. Is there something I need to configure in order for this to work? Or does it seem more like i've not got something set up correctly?

Many thanks in advance!

**Comment by freekmurze** [comment, marked as answer]
This has been fixed in a later release. Please make sure you are using the latest version.

If it continues to error, create an issue. Make sure to mention the stack trace of the error.

**Reply by mattostromhall** [reply]
Ah ok amazing, sorry should have updated 🤦 Thanks for the super quick reply, all working now, loving the package!
